Formula/n/neovim.rb

```diff
@@ -52,8 +52,6 @@ class Neovim < Formula
     regex(/^v?(\d+(?:\.\d+)+)$/i)
   end
 
-  no_autobump! because: :requires_manual_review
```

Comment:
It still requires manual review, btw. The upcoming 0.12 release will definitely need resource adjustments.

Reply:
that is fine, it would show the resource adjustment needed. (this autobump is mostly for formulae cannot be autobumped at all)